deps/build.jl

```diff
@@ -0,0 +1,19 @@
+using BinDeps
+
+@BinDeps.setup
+
+libproj = library_dependency("proj", aliases = ["libproj"])
+
+@linux_only begin
+    provides(AptGet, "libproj-dev", libproj, os = :Linux)
```

Comment:
Maybe we should be installing only the binary dependencies - that would be "libproj0" rather than "libproj-dev"